datahub/user/company_list/serializers.py

```diff
@@ -85,6 +85,21 @@ def validate_company(self, company):
 
         return company
 
+    def validate(self, data):
+        """
+        Raise a validation error if anything else other than allowed fields is updated.
+        """
+        if self.partial and self.instance:
+            allowed_fields = ['status']
+            fields = list(data.keys())
+            field_delta = list(set(fields) - set(allowed_fields))
+            if field_delta:
```

Comment:
This could be tidied up a little bit:
```suggestion
            allowed_fields = {'status'}
            extra_fields = data.keys() - allowed_fields
            if extra_fields:
```